src/main/java/com/tira/restaurants/handler/CustomAuthenticationFailureHandler.java

```diff
@@ -0,0 +1,50 @@
+package com.tira.restaurants.handler;
+
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.WebAttributes;
+import org.springframework.security.web.authentication.SimpleUrlAuthenticationFailureHandler;
+import org.springframework.stereotype.Component;
+
+import com.tira.restaurants.service.LoginAttemptService;
+
+@Component("authenticationFailureHandler")
+public class CustomAuthenticationFailureHandler extends SimpleUrlAuthenticationFailureHandler {
+
+	@Autowired
+	LoginAttemptService loginAttemptService;
+
```

Comment:
Handler for Authentication Failure, I will probably modify this part as well, since I don't want to block users by IP adress.